src/stk/molecular/molecules/building_block.py

```diff
@@ -610,24 +615,28 @@ def _init_from_rdkit_mol(
             placer_ids=placer_ids,
             functional_groups=self._functional_groups,
         )
+        self._core_ids = self._get_core_ids(
+            functional_groups=self._functional_groups,
+        )
 
-    def _normalize_placer_ids(self, placer_ids, functional_groups):
+    def _normalize_placer_ids(
+        self,
+        placer_ids: tuple[int, ...],
+        functional_groups: Iterable[FunctionalGroup],
+    ) -> frozenset[int]:
         """
         Get the final *placer* ids.
 
-        Parameters
-        ----------
-        placer_ids : :class:`tuple` of :class:`int`
-            The ids of *placer* atoms or ``None``.
+        Parameters:
 
-        functional_groups : :class:`iterable`
-            The :class:`.FunctionalGroup` instances of the building
-            block.
+            placer_ids: The ids of *placer* atoms or ``None``.
```

Comment:
If placer ids can be then, then the type hint is wrong, and should be `Optional[tuple[int, ...]]`

Reply:
Hehe, check again. 
hint: ...

Reply:
Ahh forgot to change this back after fixing it and breaking it.